doc/db.sql

```diff
@@ -56,7 +56,7 @@ CREATE INDEX idx_tracks_popularity ON tracks(popularity);
 CREATE TABLE IF NOT EXISTS repository (
     track_id UUID REFERENCES tracks(track_id),
     source_id UUID REFERENCES datasources(source_id),
-    link VARCHAR(512) UNIQUE NOT NULL,
+    link VARCHAR(512) NOT NULL,
```

Comment:
Yes, I found a situation on the different albums but these name of tracks are equaled. And now, the search keyword jus applies the name of artist and track, so the link of the searching result will not be unique.

I had tried to apply the name of artist, album, and track to the search keywords, sometimes there is no result found or the accuracy was bad.

Maybe we can discuss the better solution in the future.